examples/send_dates_transaction.js

```diff
@@ -0,0 +1,45 @@
+const pkg = require('../package.json');
+const { Exporter } = require('../index')
+const exporter = new Exporter(pkg.name, true)
+
+async function pushData() {
+  const timestamp = Math.floor(new Date() / 1000)
```

Comment:
nitpick: I'm not sure how the implicit type conversion is handled here. If you want to not use type coercion you can do: `Math.floor(new Date().getTime() / 1000)`

Reply:
Good point, changed.